api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -1892,6 +1892,54 @@ message SFTP {
   string Error = 12 [(gogoproto.jsontag) = "error,omitempty"];
 }
 
+// SFTPSummary is emitted at the end of an SFTP transfer
+message SFTPSummary {
```

Comment:
How can I tell if this was a upload or download? Looks at bytes read vs written?

Reply:
Yep, technically you can write and read to the same file handle so classifying isn't as straightforward as SCP. But for 99% of cases the program that is issuing SFTP requests will likely only be reading or writing to files, so I can label this if needed